gremlin-go/driver/client.go

```diff
@@ -71,25 +73,51 @@ func NewClient(url string, configurations ...func(settings *ClientSettings)) (*C
 		transporterType: settings.TransporterType,
 		connection:      conn,
 	}
+	// TODO: PoolSize must be 1 on Session mode
 	return client, nil
 }
 
 // Close closes the client via connection.
 func (client *Client) Close() error {
```

Comment:
Remove error from return signature and just return at end.

Reply:
yep just pushing that now